specs/agents/universal-profiling-integration.md

```diff
@@ -179,19 +179,19 @@ All messages have the following layout:
 
 ## Profiler Registration Message
 
-Whenever the profiling host agent starts communicating for the first time with a process running an APM Agent, it MUST send this message.
-This message is used to let the APM-agent know that a profiler is actually active on the current host. Note that an APM-agent may receive this message zero, one or several times: this may happen if no host agent is active, if one is active or if a host agent is restarted during the lifetime of the APM-agent respectively.
+Whenever the profiling agent starts communicating for the first time with a process running an APM Agent, it MUST send this message.
+This message is used to let the APM-agent know that a profiler is actually active on the current host. Note that an APM-agent may receive this message zero, one or several times: this may happen if no profiling agent is active, if one is active or if a profiling agent is restarted during the lifetime of the APM-agent respectively.
 
-The *message-type* is `2` and the current *minor-version* is `1`.
+The *message-type* is `2` and the current *minor-version* is `2`.
```

Comment:
```suggestion
The *message-type* is `2` and the current *minor-version* is `1`.
```

Clarifying that the `host-id` might be an empty string is imo more a clarification of the existing format and doesn't have any impact on the decoding of the message, therefore let's just keep it at minor version `1`.
Also this feature is currently only implemented for the java agent, giving us this flexibility to change the spec more leniently.